src/auxkernels/MFEMDivAux.C

```diff
@@ -0,0 +1,41 @@
+#pragma once
+#include "MFEMDivAux.h"
+
+registerMooseObject("PlatypusApp", MFEMDivAux);
+
+/*
+Class to set an L2 auxvariable to be the divergence of a H(div) vector variable.
+*/
+InputParameters
+MFEMDivAux::validParams()
+{
+  InputParameters params = MFEMAuxKernel::validParams();
+  params.addClassDescription(
+      "Calculates the divergence of a H(div) conforming RT source variable and stores the result"
+      " on a L2 conforming result auxvariable");
+  params.addRequiredParam<VariableName>("source",
+                                        "Vector H(div) MFEMVariable to take the divergence of.");
+  params.addParam<mfem::real_t>("scale_factor", 1.0, "Factor to scale result auxvariable by.");
+  return params;
+}
+
+MFEMDivAux::MFEMDivAux(const InputParameters & parameters)
+  : MFEMAuxKernel(parameters),
+    _source_var_name(getParam<VariableName>("source")),
+    _source_var(*getMFEMProblem().getProblemData()._gridfunctions.Get(_source_var_name)),
+    _scale_factor(getParam<mfem::real_t>("scale_factor")),
+    _hdiv_fespace(*_source_var.ParFESpace()),
+    _l2_fespace(*_result_var.ParFESpace()),
```

Comment:
Do these actually need to be stored as class members? Can't you just access them to construct the curl operator, without saving references to them?

Reply:
We can indeed - thanks, removed these